scarlet/bbox.py

```diff
@@ -45,34 +39,24 @@ def from_image(image):
         return Box(image.shape)
 
     @staticmethod
-    def from_bounds(front, back, bottom, top, left, right):
+    def from_bounds(*args):
         """Initialize a box from its bounds
 
         Parameters
         ----------
-        bottom: int
-            Minimum in the y direction.
-        top: int
-            Maximum in the y direction.
-        left: int
-            Minimum in the x direction.
-        right: int
-            Maximum in the x direction.
+        args: ints
+            Min/Max coordinate for every dimension
 
         Returns
         -------
         bbox: :class:`scarlet.bbox.Box`
             A new box bounded by the input bounds.
         """
-        if back < front:
-            back, front = front, back
-        if top < bottom:
-            top, bottom = bottom, top
-        if right < left:
-            right, left = left, right
-        return Box(
-            (back - front, top - bottom, right - left), origin=(front, bottom, left)
-        )
+        assert len(args) % 2 == 0
+        dims = len(args) // 2
```

Comment:
SHouldn't we check `dims == len(self.shape)`?

Reply:
This is a static constructor, there is no `self.<anything>` yet.

Reply:
`self` doesn't exist yet. that's a `classmethod`.